src/components/ConnectorTile.tsx

```diff
@@ -0,0 +1,318 @@
+                                    onClick={() => {
+                                        if (
+                                            row.connector_tags[0].protocol ===
+                                            'capture'
+                                        ) {
+                                            navigate(
+                                                getPathWithParam(
+                                                    authenticatedRoutes.captures
+                                                        .create.fullPath,
+                                                    authenticatedRoutes.captures
+                                                        .create.params
+                                                        .connectorID,
+                                                    row.connector_tags[0].id
+                                                )
+                                            );
+                                        } else if (
+                                            row.connector_tags[0].protocol ===
+                                            'materialization'
+                                        ) {
+                                            navigate(
+                                                getPathWithParam(
+                                                    authenticatedRoutes
+                                                        .materializations.create
+                                                        .fullPath,
+                                                    authenticatedRoutes
+                                                        .materializations.create
+                                                        .params.connectorId,
+                                                    row.connector_tags[0].id
+                                                )
+                                            );
+                                        }
+                                    }}
+                                >
+                                    {row.connector_tags[0].protocol ===
+                                    'capture' ? (
+                                        <FormattedMessage id="connectorTable.actionsCta.capture" />
+                                    ) : (
+                                        <FormattedMessage id="connectorTable.actionsCta.materialization" />
+                                    )}
+                                </Button>
+                            </Stack>
+                        </Paper>
+                    </Grid>
+                ))
+            ) : (
+                <Grid item sx={{ width: '100%' }}>
+                    {isValidating ||
+                    tableState.status === TableStatuses.LOADING ? (
+                        <Box>
+                            <LinearProgress />
+                        </Box>
+                    ) : (
```

Comment:
I'm not totally sure on how this would impact but I wonder if we want to move some loading stuff like this to not totally remove the view and just display this in addition to a grayed out view.

Reply:
I will look into this separately.